src/ServiceControl.Config/Commands/UpgradeServiceControlInstanceCommand.cs

```diff
@@ -151,7 +151,7 @@ public override async Task ExecuteAsync(InstanceDetailsViewModel model)
                 }
             }
 
-            if (!instance.AppConfig.AppSettingExists(SettingsList.DatabaseMaintenancePort.Name))
+            if (!instance.AppConfig.AppSettingExists(AuditInstanceSettingsList.DatabaseMaintenancePort.Name))
```

Comment:
Should this be AuditInstanceSettingsList? All of the other options in the file are ServiceControlSettings.